airflow/providers/amazon/aws/sensors/s3.py

```diff
@@ -40,15 +41,16 @@
 
 class S3KeySensor(BaseSensorOperator):
     """
-    Waits for a key (a file-like instance on S3) to be present in a S3 bucket.
+    Waits for one or multiple keys (a file-like instance on S3) to be present in a S3 bucket.
```

Comment:
can you also make this edit in `S3KeySizeSensor` ?

Reply:
`S3KeySizeSensor` inherits from `S3KeySensor` so now that you are changing `bucket_key` from `str` to `Union[str, List[str]]` it means that user will now be able to pass list of keys to `S3KeySizeSensor`.

If this is not right for the use case of `S3KeySizeSensor` then it require some additional changes.

Reply:
Oh you're right. Let me do that change

Reply:
Done